installers/rpm-code/codes/flash.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+flash_main() {
+    codes_dependencies hypre
+    codes_download_proprietary "flash/FLASH-4.6.2.tar.gz" "FLASH4.6.2"
+    local flash_src_root=$PWD
+    patch_makefile
+    for n in CapLaser RTFlame
+    do
+        setup_$n $n
+        make_and_install $n
+    done
+}
+
+make_and_install() {
+    cd $1
+    codes_make
+    install -m 755 flash4 "${codes_dir[bin]}"/flash4-$1
+    install -d 755 "${codes_dir[share]}"/flash4
+    install -m 444 setup_units "${codes_dir[share]}"/flash4/setup_units-$1
+    cd ..
+}
+
+patch_makefile() {
+    patch sites/Prototypes/Linux/Makefile.h <<'EOF'
+6c6
+< MPI_PATH   = /usr/local/mpich2/
+---
+> MPI_PATH   = /usr/lib64/mpich/
+8,9c8,9
+< HDF5_PATH  = /usr/local/hdf5
+< HYPRE_PATH = /usr/local/hypre
+---
+> HDF5_PATH  = /usr/lib64/mpich
+> HYPRE_PATH = /home/vagrant/.local
+157c157
+< #----------------------------------------------------------------------------
+---
+> #----------------------------------------------------------------------------
+EOF
+}
+
+setup_CapLaser() {
+    cd source/Simulation/SimulationMain/magnetoHD
+    codes_download_proprietary "flash/CapLaserBELLA-4.6.2.tar.gz" "CapLaserBELLA"
+    cd $flash_src_root
+    ./setup -auto magnetoHD/CapLaser -2d -nxb=16 -nyb=16 +hdf5typeio \
```

Comment:
Does CapLaser need to be magnetoHD? What happens if you make it exactly like RTFlame, and put it one level up? Are there includes in the CaplaserBELLA that are relative to the directory (or are they using `cc -I` paths).

Reply:
https://github.com/radiasoft/download/pull/95/commits/4b4294d531035f961f3421d9e7450d2b8d2c07b2